router/router.nginx.conf.in

```diff
@@ -75,8 +75,8 @@ server {
         include /etc/nginx/proxy.conf;
     }
 
-    listen 80;
-    listen [::]:80;
+    listen 443 ssl;
```

Comment:
Does this change need to be here?

Reply:
Yes. Talking to `hello` has been broken since we switched the router to TLS. This is for people speaking to the router. Notice that in the proxy block above, hello is still using `http`. So it's HTTPS between anyone (including my new test) and router, then HTTP between router and hello.